packages/lib/storage/src/new/write.test.ts

```diff
@@ -0,0 +1,37 @@
+import { addConversation, addMessage } from './write';
+import { db, makeEnvelop, testEnvelop } from './testHelper';
+
+describe('write', () => {
+    describe('addConversation', () => {
+        it('should add a conversation to the database', async () => {
+            const contactEnsName = 'c2';
+
+            expect(await addConversation(contactEnsName, db)).toEqual({
+                conversationList: {
+                    key: '0x9c7f05ff5af8632a1b4f9112e3451e265696646ad70187d083282b3d8886be88',
```

Comment:
can we generate the keys dynamically so it is understandable what actually is tested here